src/qt/transactionrecord.cpp

```diff
@@ -411,7 +411,11 @@ QList<TransactionRecord> TransactionRecord::decomposeTransaction(const interface
             if (wtx.txout_address_is_mine[i]) {
                 // Received by Bitcoin Address
                 sub.type = TransactionRecord::RecvWithAddress;
-                sub.address = EncodeDestination(wtx.txout_address[i]);
+                bool fBech32 = false;    
+                if (boost::get<CScriptID>(&wtx.txout_address[i])){
+                    fBech32 = true;
+                }            
```

Comment:
Can this be condensed into a single line instead of the if? say, something like
`bool fBech32 = (bool)boost::get<CScriptID>(...);` or `boost::get<CScriptID>(...) != ...` ?

Reply:
That's true:
`bool fBech32 = boost::get<CScriptID>(&wtx.txout_address[i])` would do the trick; but functionally it's the same and and it's not against style; so I'll say that's a NIT

Another NIT: space between the end of the if and the `{`

But given problems with this problem, I'm not concerned with holding it up.